src/test/java/com/verisign/storm/metrics/GraphiteMetricsConsumerTest.java

```diff
@@ -184,17 +210,16 @@ public void shouldReadDataAndSendLong(Collection<DataPoint> dataPoints) {
     return new Object[][] { new Object[] { dpList } };
   }
 
-  @Test(dependsOnMethods = { "shouldReadConfigAndContext" }, dataProvider = "generateDataPointsFloat")
+  @Test(dependsOnMethods = { "shouldInitializeGraphiteReporter" }, dataProvider = "generateDataPointsFloat")
```

Comment:
I think we can postpone integration tests for the time being (although they would be nice to have of course).